packages/bpk-component-page-indicator/README.md

```diff
@@ -0,0 +1,38 @@
+# bpk-component-page-indicator
+
+> Backpack example component.
+
+## Installation
+
+```sh
+npm install bpk-component-page-indicator --save-dev
+```
+
+## Usage
+
+```js
+import React from 'react';
+import BpkPageIndicator from 'bpk-component-page-indicator';
+
+export default () =>
+  <BpkPageIndicator
+    currentItem={0}
+    totalItems={7}
+    updateItemCallback={(newIndex) => {
+      console.log('Slide to new index:', newIndex);
+    }}
+  />;
+```
+
+## Props
+
+| Property            | PropType | Required | Default Value |
+|---------------------|----------|----------|---------------|
+| currentItem         | number   | true     | -             |
+| totalItems          | number   | true     | -             |
```

Comment:
We can use `currentIndex` and `totalBullets` instead

Reply:
Have changed name from currentItem -> currentIndex, and totalItems -> totalBullets, thank you!